README.md

```diff
@@ -5,26 +5,28 @@
   <img alt="A badge displaying the number of messages being sent via the project" src="https://img.shields.io/badge/dynamic/json?label=Messages%20Sent&query=totalChatShort&url=https%3A%2F%2Faidn5.com%2Fstats.json">
   <a href="https://discord.gg/ej7tQHPF8y"><img src="https://img.shields.io/discord/1002575659694043206?color=5865F2&logo=discord&logoColor=white" alt="Discord server" /></a>
 </p>
-Service connects multiple guilds and discord server together  
-This project is made to be fully flexible and customizable
-offering quality user experience while keeping it simple
 
-> **Disclaimer: This project interacts with Hypixel in an unintended way by simulating a minecraft client and processes
-> packets which might get you banned if you over-abuse it too much.  
+## Introduction
+
+A service that connects multiple Hypixel guilds and Discord servers together.
+This project is made to be fully flexible and customisable, offering a high quality user experience while keeping it simple.
+
+> **_DISCLAIMER_: This project interacts with Hypixel in an unintended way by simulating a minecraft client and by processing
+> packets which might get you banned if over-abused too much.  
 > Just like any other modification and service that interacts with Hypixel servers, this goes without saying: "Use at
 > your own risk"**
 
 ## Migrating To 3.x.x
 
 Breaking changes:
 
-- `config.yaml` options has been moved around. Check the new `config_example.yaml` and compare it with the old config file
-- Version 3.x websocket is incompatible with older versions. Cannot connect the new version with the old one
-- Chat Commands have been moved to their own instance in `config.yaml`. Only enable one instance when using websocket
-- Project have moved to ESM. Old plugins that still use commonjs will not work anymore
-- Minecraft bot and websocket have been stripped down to bare metal. Any plugin that use those features must update their code.
-- Punishment system has been revised and now uses `./configs/` to save its state as a file. Directory is now git-ignored.
-- any unexpected error during the launching phase will result in a shutdown with an exit code instead of sending a notification
+- `config.yaml` has been reformatted. Check the new `config_example.yaml` and compare it with the old config file.
+- Version 3.x websocket is incompatible with older versions. You cannot connect the new version with the old one.
+- Chat Commands have been moved to their own instance in `config.yaml`. Only enable one instance when using websocket.
+- Project has been moved to ESM. Old plugins that still use commonjs will not work anymore.
+- The Minecraft bot and websocket have been stripped down to bare metal. Any plugins that use those features must be updated accordingly.
+- The punishment system has been revised and now uses `./configs/` to save it's state as a file. The `./configs/` directory is now git-ignored.
```

Comment:
`its` too I believe.

Reply:
Fixed instances of "it's" being contracted, not possessive